TLM/TLM/UI/SubTools/SpeedLimits/Overlay/OverlayLaneSpeedlimitHandle.cs

```diff
@@ -135,30 +135,28 @@ namespace TrafficManager.UI.SubTools.SpeedLimits.Overlay {
         /// <param name="segmentList">input list of roundabout segments (must be oneway, and in the same direction).</param>
         /// <param name="segmentId0">The segment to match lane agaisnt.</param>
         /// <param name="sortedLaneIndex">Index.</param>
-        internal IEnumerable<LanePos> FollowRoundaboutLane(List<ushort> segmentList,
-                                                           ushort segmentId0,
-                                                           int sortedLaneIndex) {
+        internal IEnumerable<LanePos> FollowRoundaboutLane(
+            List<ushort> segmentList,
+            ushort segmentId0,
+            int sortedLaneIndex) {
+
             bool invert0 = segmentId0.ToSegment().m_flags.IsFlagSet(NetSegment.Flags.Invert);
-            ExtSegmentManager extSegmentManager = ExtSegmentManager.Instance;
 
-            int count0 = extSegmentManager.GetSortedLanes(
-               segmentId: segmentId0,
-               segment: ref segmentId0.ToSegment(),
-               startNode: null,
-               laneTypeFilter: SpeedLimitManager.LANE_TYPES,
-               vehicleTypeFilter: SpeedLimitManager.VEHICLE_TYPES,
-               sort: false).Count;
+            int count0 = segmentId0.ToSegment().GetSortedLanes(
```

Comment:
Double `ToSegment()` call

Reply:
I wasn't sure what to do here because we can't store ref in an enumerable.

Reply:
Oh, ok true, I didn't check in the IDE, ignore